rocAL/rocAL_pybind/amd/rocAL/plugin/pytorch.py

```diff
@@ -29,14 +29,14 @@ def __next__(self):
 
         self.loader.copyImage(self.out_image)
         if((self.loader._name == "Caffe2ReaderDetection") or (self.loader._name == "CaffeReaderDetection")):
-         
+
             for i in range(self.bs):
                 size = b.getImageNameLen(self.loader._handle,i)
                 print(size)
                 self.array = np.array(["                 "])
```

Comment:
This is not a good way to define array with spaces??

Reply:
Changed it